### ISSUE TITLE
Add a "Learn More about Clubs" section to the site

### ISSUE THREAD

**Issue by iamjessklein** [mozilla/teach.webmaker.org#428]
This should take into account [the sitemap](http://invis.io/9G2DK7SR2).

**Comment by hannahkane** [comment]
Some context about where this came from originally:
 * During earlier conversations we'd discussed the idea of having people indicate their interest in starting a Club on the map, so that others could see if there was interest near them, and get in touch with potential collaborators.
 * We nixed that idea for v1 since it didn't seem urgent (and there's a bit of an empty shelf problem; it's not that interesting unless there's A LOT of interested people in all parts of the world), and opted to go with a simple contact us form
 * We ended up cutting that from the v1 as well, since it was competing with too many things on the Clubs page

Some things that I think are important for our implementation moving forward:
 * Whether we implement the full-fledged idea described above or not, I think this needs to be connected specifically to the Clubs content, rather than the site as a whole. The purpose is to offer people a soft entry point into the world of Clubs—it's not just a Contact Us form.
 * We likely need a different long-term solution for the variety of Clubs-related content we're offering. We can probably divide our audience into two groups: those who have registered a Club, and those who haven't yet. Content like the Clubs Curriculum and the Toolkit are predominantly for the first audience, whereas the CTAs related to adding a Club or learning more are for the uninitiated. :)
